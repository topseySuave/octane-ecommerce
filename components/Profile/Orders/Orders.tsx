import { Typography } from 'antd';
import { ProductCardHorizontal } from 'components/ui/ProductCard';

const { Title } = Typography;

const Orders = () => {
  return (
    <>
      <Title level={3}>Order(s)</Title>
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
    </>
  );
};

export default Orders;
