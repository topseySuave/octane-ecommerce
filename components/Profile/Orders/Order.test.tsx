import { shallow } from 'enzyme';
import React from 'react';

import Orders from './Orders';

describe('Orders', () => {
  it('Renders with user profile tabs settings', () => {
    const tree = shallow(<Orders />);
    expect(tree).toMatchSnapshot();
    // const mounted = shallow(<Profile />);
    // const profile = mounted.find('content');
    // console.log(profile);
  });
});
