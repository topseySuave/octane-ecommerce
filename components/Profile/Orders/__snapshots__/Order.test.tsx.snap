// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders Renders with user profile tabs settings 1`] = `ShallowWrapper {}`;
